exercises/practice/sublist/.docs/instructions.append.md

```diff
@@ -0,0 +1,6 @@
+# About Enum
+Use Enum values to show relationship between two list, e.g. SUBLIST
+
+To know more about Enum in powershell please check out [Enum module.](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_enum)
+
+You can also learn and practice it from a previous exercise [Triangle.](https://exercism.org/tracks/powershell/exercises/triangle)
```

Comment:
I think this is not line is not needed, but otherwise I think everything looks good.